machine_translation/prepare_data.py

```diff
@@ -160,7 +161,7 @@ def merge_parallel(src_filename, trg_filename, merged_filename):
                 while True:
                     lline = left.readline()
                     rline = right.readline()
-                    if (lline == '') or (rline == ''):
+                    if (lline == '') and (rline == ''):
```

Comment:
this is not the thing that is causing the crash, thats why i removed that section from PR comment